README.md

```diff
@@ -29,23 +29,43 @@ Install [LaTex](https://www.latex-project.org/get/). If you are on Linux, then y
     \input{<file_name>.tex}
 ```
 
-### Build PDF
+### Build the PDF
 
-1. After writing your LaTex document, build it from project directory using:
+#### Method 1: Use the makefile
+
+After writing your LaTex document, give command `make` from the project directory.
+
+```shell
+    make
 ```
-    pdflatex your_project.tex
+
+And your `main.pdf` shall be good to go!
+
+__NOTE__:
+- Every time you add/remove a reference you need to re-generate the PDF with `make` command.
+
+#### Method 2: Build PDF the hard way 
+
+1. After writing your LaTex document, build it from project directory using:
+```shell
+    pdflatex main
 ```
 
 2. In case your document contains references (citations/images/links/etc) use:
-```
-    bibtex your_project.aux
-    pdflatex your_project.tex
-    pdflatex your_project.tex
+```shell
+    bibtex main
+    python enumerate-refs.py
+    pdflatex main
+    pdflatex main
 ```
 
 __NOTE__:
-- Every time you add a new reference, you need to run these 3 commands.
-- *.aux* file is generated automatically after using `pdflatex` for the initial build.
+- Every time you add/remove a reference you need to re-generate the PDF. For which either run command `make` as described [above](#method-1-use-the-makefile), OR do as follows:
+```shell
+    rm main.bbl
```

Comment:
You can just mention this one command.

Also, I don't think it is necessary to remove these files if I want to regenerate PDF according to the new changes, isn't it?

Reply:
Yes, you are right, we don't need them. I'll change.